prem_utils/models.json

```diff
@@ -593,6 +593,57 @@
                     "coming_soon": true
                 }
             ]
+        },
+        {
+            "provider": "anyscale",
+            "models": [
+                {
+                    "slug": "meta-llama/Llama-2-7b-chat-hf",
+                    "model_type": "text2text",
+                    "context_window": 4096
+                },
+                {
+                    "slug": "meta-llama/Llama-2-13b-chat-hf",
+                    "model_type": "text2text",
+                    "context_window": 4096
+                },
+                {
+                    "slug": "meta-llama/Llama-2-70b-chat-hf",
+                    "model_type": "text2text",
+                    "context_window": 4096
+                },
+                {
+                    "slug": "codellama/CodeLlama-70b-Instruct-hf",
+                    "model_type": "text2text",
+                    "context_window": 4096
+                },
+                {
+                    "slug": "mistralai/Mistral-7B-Instruct-v0.1",
```

Comment:
Double check if the slug is used in another provider.

Reply:
Yes I added the `anyscale/` prefix